docs/ui-spec.md

```
@@ -0,0 +1,2347 @@
+---
+
+# Tempo AI UI/UX 仕様書
+
+**バージョン**: 1.0  
+**最終更新日**: 2025 年 12 月 10 日  
+
+---
+
+## 目次
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)